### PR TITLE
Rebase abort

### PR DESCRIPTION
**Pull request by bpeabody**
depends on https://github.com/twosigma/git-meta/pull/150

### CODE REVIEW

**Comment by bpeabody** [comment]
Is this implicit approval for https://github.com/twosigma/git-meta/pull/150?

**Comment by novalis** [comment]
Actually, I meant to approve #150 but accidentally approved this one -- let me just review this one a little more.

File: node/lib/util/read_repo_ast_util.js [hunk 1]
```diff
@@ -411,6 +411,13 @@ exports.readRAST = co.wrap(function *(repo) {
         }
     });
 
+    const rebase = yield RebaseFileUtil.readRebase(repo.path());
+
+    if (null !== rebase) {
```
Comment (novalis) [inline node/lib/util/read_repo_ast_util.js]: Why are we loading these commits and not using them?

**Comment by novalis** [comment]
OK, actually approved.

**Pull request merged**